scripts/script.js

```diff
@@ -0,0 +1,8 @@
+window.onload=function(){
+    document.getElementById("hamburger").addEventListener("click", ()=>{
+        document.getElementById("navigation").classList.toggle("open")
+        document.getElementById("header").classList.toggle("sticky")
+        document.getElementById("hamb-icon").classList.toggle("hidden")
+        document.getElementById("close-icon").classList.toggle("hidden")
```

Comment:
Good work with applying classes for changing the element's visibility!

In general, using 'display: none;' for hiding UX elements is not recommended as it will trigger a browser repaing and elements reflow. What you could do here is to use visibility toggles instead (visibility: hidden or opacity: 0) and set the buttons for closing and expanding the menu overlapping with absolute positioning.